unicoremc/models.py

```diff
@@ -376,31 +442,31 @@ def create_pyramid_settings(self):
                 self.available_languages.all(),
                 self.default_language or Localisation._for('eng_GB'),
                 self.ga_profile_id,
-                self.hub_app()
+                self.hub_app(),
+                [repo.name() for repo in self.all_repos()]
```

Comment:
Following previous comment about `springboard-iogt` having a `unicore.own_content_repo_url`, I don't think this should be `self.all_repos()` - we currently don't include the `own_repo` in the aggregation. My thinking is that the `own_repo` with be a data store for additional content on the site rather than it being additional data to be aggregated.